### DISCUSSION TITLE
Raspberry Pi status

### DISCUSSION THREAD

**Discussion by kwankiu** [kwankiu/archlinux-installer-rock5#39] [General]
Raspberry Pi status
Whats working what’s not and what to add.

**Comment by kwankiu** [comment]
@erayrafet

**Comment by erayrafet** [comment]
Is there a new build?
The last one wouldn't run a DE without modification.

**Reply by kwankiu** [reply]
You will still need to update config.txt
I am not sure if the last one had firmware-raspberrypi properly installed. This one should have. Needs testing tho
